github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
I'm going to give it a go using https://github.com/ruby-concurrency/thread_safe, loading the entire structure into mem at start, and using an at_exit block to save. Downside is that interrupted runs might lose the entire cache, but PStore negated the entire idea of the cache.